pyfaidx/__init__.py

```diff
@@ -293,7 +293,7 @@ def __init__(self, filename, default_seq=None, key_function=lambda x: x.split()[
         self.filename = filename
 
         filenameLower = filename.lower()
-        if filenameLower.endswith('.bgz'):
+        if filenameLower.endswith('.bgz') or filenameLower.endswith('.gz'):
```

Comment:
Can we just do `filename.lower().endswith('...')` instead of creating a new local variable?